.github/workflows/simulation-1.2.yml

```diff
@@ -71,5 +71,21 @@ jobs:
       run: |
         ./script/test unit
         ./script/test cert_suite tests/scripts/thread-cert/v1_2_*
+    - name: Keep-1-2-only
+      run: |
+        ./script/test tar 1.1
+        ./script/test tar 1.2-bbr
+    - name: Codecov
+      uses: codecov/codecov-action@v1
```

Comment:
Have you checked the arguments of  `Codecov` to see if there is any argument we can use to address this problem?

Reply:
I found some documents and started an [issue](https://github.com/codecov/codecov-action/issues/106) at `codecov-action`, but unfortunately for now I didn't get anyhing that helps. Anyway I would follow up this question.

Reply:
> I found some documents and started an [issue](https://github.com/codecov/codecov-action/issues/106) at `codecov-action`, but unfortunately for now I didn't get anyhing that helps. Anyway I would follow up this question.

Have you tried the directory parameter? I suspect that it might solve our problem.

Reply:
I couldn't find the related document. Is there a document showing how to use the parameter?

Reply:
I think it's not the one we want. It's the directory to search for coverage reports, e.g., generated one. What we want is to specify the build directory to search for `*.gcno` and `.gcda` which are used to generate the reports.